rb/spec/integration/selenium/webdriver/bidi/log_inspector_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require_relative '../spec_helper'
+
+module Selenium
+  module WebDriver
+    class BiDi
+      describe LogInspector, exclusive: {browser: %i[chrome firefox]} do
```

Comment:
Doesn't `edge` also support it?

Reply:
It's less intuitive than I want, but was the best I could come up with for guards:

* `except` - bug is present on specified configurations (runs non-specified configurations and fails if they pass)
* `only` -  bug is present on non-specified configurations (runs specified configurations and fails if they pass)
* `exclusive` - feature only applies to specified configurations
* `exclude` - feature is valid for specified configurations, but bug is present on it and it causes test suite to crash

The way I use these, if it is a DriverExtension and the methods are only imported to certain browsers, use `exclusive` on top level `describe`, then `except`/`only` on sub-levels (`exclude` if it breaks something when using the others)

This feature is not an extension, the method is available on all drivers. So use `only` for everything where it actually works, or `except` with `reason` values for everything that doesn't work.